packages/mdc-snackbar/README.md

```diff
@@ -108,69 +131,173 @@ This will move the snackbar offscreen until the CSS is fully loaded and avoids a
+`labelText` | `string` | Gets/sets the `textContent` of the label element.
+`actionButtonText` | `string` | Gets/sets the `textContent` of the action button element.
 
-Method Signature | Description
---- | ---
-`show(data: DataObject=) => void` | Displays the snackbar. `data` populates the snackbar and sets options (see below).
+> **NOTE**: Setting `labelText` while the snackbar is open will cause screen readers to announce the new label. See [Screen Readers](#screen-readers) below for more information.
 
-### DataObject Properties
+### `MDCSnackbar` Methods
 
- Property | Type | Description
---- | --- | ---
- `message` | string | Mandatory. The text message to display.
- `timeout` | number | The amount of time in milliseconds to show the snackbar. Defaults to `2750`.
- `actionHandler` | function | The function to execute when the action is clicked.
- `actionText` | string | Mandatory if `actionHandler` is set. The text to display for the action button.
- `multiline` | boolean | Whether to show the snackbar with space for multiple lines of text.
- `actionOnBottom` | boolean | Whether to show the action below the multiple lines of text (only applicable when `multiline` is true).
+Method Signature | Description
+--- | ---
+`open() => void` | Opens the snackbar.
+`close(reason: string=) => void` | Closes the snackbar, optionally with the specified reason indicating why it was closed.
 
 ### Events
 
 Event Name | `event.detail` | Description
 --- | --- | ---
-`MDCSnackbar:hide` | `{}` | Emitted when the Snackbar is hidden.
-`MDCSnackbar:show` | `{}` | Emitted when the Snackbar is shown.
+`MDCSnackbar:opening` | `{}` | Indicates when the snackbar begins its opening animation.
+`MDCSnackbar:opened` | `{}` | Indicates when the snackbar finishes its opening animation.
+`MDCSnackbar:closing` | `{reason: ?string}` | Indicates when the snackbar begins its closing animation. `reason` contains the reason why the snackbar closed (`dismiss` or `action`).
+`MDCSnackbar:closed` | `{reason: ?string}` | Indicates when the snackbar finishes its closing animation. `reason` contains the reason why the snackbar closed (`dismiss` or `action`).
 
-## Usage Within Frameworks
+### Usage Within Frameworks
 
-If you are using a JavaScript framework, such as React or Angular, you can create a Snackbar for your framework. Depending on your needs, you can use the _Simple Approach: Wrapping MDC Web Vanilla Components_, or the _Advanced Approach: Using Foundations and Adapters_. Please follow the instructions [here](../../docs/integrating-into-frameworks.md).
+If you are using a JavaScript framework, such as React or Angular, you can create a Snackbar for your framework. Depending on your needs, you can use the _Simple Approach: Wrapping MDC Web Vanilla Components_, or the _Advanced Approach: Using Foundations and Adapters_. Please see [Integrating MDC Web into Frameworks](../../docs/integrating-into-frameworks.md).
```

Comment:
See comment RE separating this wording change to apply across all READMEs in this format

Reply:
Done